lib/document-to-html.js

```diff
@@ -145,6 +145,9 @@ function DocumentConversion(options, comments) {
         if (run.isBold) {
             paths.push(findHtmlPathForRunProperty("bold", "strong"));
         }
+        if (run.isSmallCaps) {
+            paths.push(findHtmlPathForRunProperty("smallCaps"));
+        }
```

Comment:
I might just be missing it, but I think this is missing the corresponding test cases?